app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::CommentsController < ApplicationController
+  def index
+    render json: Comment.all
```

Comment:
This is not quite right. Because these comments are associated with an individual player, you'll want this response to be something more like `Comment.where(player_id: params[:player_id])`